--- libhif-0.1.8/libhif/hif-context.c.orig	2015-03-17 21:14:28.322673463 +0100
+++ libhif-0.1.8/libhif/hif-context.c	2015-03-17 21:19:36.569327195 +0100
@@ -36,6 +36,7 @@
 
 #include <hawkey/query.h>
 #include <hawkey/packagelist.h>
+#include <hawkey/version.h>
 #include <librepo/librepo.h>
 
 #include "hif-cleanup.h"
@@ -917,6 +918,9 @@
 	solv_dir_real = hif_realpath (priv->solv_dir);
 	priv->sack = hy_sack_create (solv_dir_real, NULL,
 				     priv->install_root,
+#if HY_VERSION_CHECK(0,5,3)
+				     NULL,
+#endif
 				     HY_MAKE_CACHE_DIR);
 	if (priv->sack == NULL) {
 		g_set_error (error,
